src/modules/completedTask.js

```diff
@@ -24,6 +24,7 @@ const completedTask = () => {
   const clearAllCompleted = document.querySelector('.clear-button');
   clearAllCompleted.addEventListener('click', () => {
     clearCompletedTasks();
+    window.location.reload();
   });
 };
```

Comment:
[OPTIONAL]
- [ ] Using `window.location.reload` anytime you make changes to the DOM is an efficient way. I recommend you find another way to make changes to the DOM without always reloading the page. This is true for adding and removing tasks. For instance, you could have a function that creates the ToDo list dynamically anytime you add or remove a task. With that, there will be no need to hard reload every time changes are made.

Reply:
@cwaku
Noted with thanks.